pkg/config/config.go

```diff
@@ -0,0 +1,112 @@
+package config
+
+import (
+	"strconv"
+
+	"github.com/pelletier/go-toml"
+)
+
+// DefaultConfig for the DC/OS CLI.
+//
+// All config defaults are their zero value except for the timeout which is 180 seconds.
+var DefaultConfig = Config{
```

Comment:
is the intent that some caller might want to change the global default here? if that's not critical, then please don't use a `var` and consider instead defining this as a func that returns a `Config`

Reply:
I've added a comment in the first commit, this also doesn't seem like a hard suggestion from dep (https://github.com/golang/dep/blob/master/docs/FAQ.md#should-i-commit-my-vendor-directory).

Reply:
It is not, I've moved it to a function.